codes.json

```diff
@@ -7,6 +7,56 @@
       "en": "Your access token has expired. You can request a new access token with refresh token. "
     }
   },
+  "AlreadyPartner": {
+    "status": 400,
+    "code": 4090037,
+    "error": "already_partner",
+    "description": {
+      "en": "User is already a partner."
+    }
+  },
+  "ArgumentsMissedContactForm": {
+    "error": "arguments_missed_contact_form",
+    "name": "Error",
+    "code": 4090032,
+    "description": {
+      "en": "All the arguments are required: name, email, reason, message"
+    },
+    "status": 400
+  },
+  "ArgumentsMissedDeletePartnership": {
+    "error": "arguments_missed_delete_partnership",
+    "name": "Error",
+    "code": 4090031,
+    "description": {
+      "en": "At least you need pass one of this arguments: org, token"
+    },
+    "status": 400
+  },
+  "AttributesDoNotMatch": {
+    "status": 400,
+    "code": 4000024,
+    "error": "missing_src_mapping",
+    "description": {
+      "en": "The columns of this file don't match the ones for this group."
+    }
+  },
+  "CanNotInviteYourself": {
+    "status": 400,
+    "code": 4090039,
+    "error": "can_not_invite_yourself",
+    "description": {
+      "en": "Can not invite yourself."
+    }
+  },
+  "CardDeclined": {
+    "status": 402,
+    "code": 8,
```

Comment:
Actually I was using `grunt new` as stated in the documentation here! https://github.com/venzee/venzee-errors?!